vendor.conf

```diff
@@ -27,14 +27,18 @@ github.com/imdario/mergo                            7c29201646fa3de8506f70121347
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
 # buildkit
-github.com/moby/buildkit                            f5a55a9516d1c6e2ade9bec22b83259caeed3a84 
+
+# WIP WIP WIP using branch used for https://github.com/moby/buildkit/pull/1107
+github.com/moby/buildkit                            containerd-update https://github.com/tonistiigi/buildkit.git
 github.com/tonistiigi/fsutil                        3bbb99cdbd76619ab717299830c60f6f2a533a6b
 github.com/grpc-ecosystem/grpc-opentracing          8e809c8a86450a29b90dcc9efbf062d0fe6d9746
 github.com/opentracing/opentracing-go               1361b9cd60be79c4c3a7fa9841b3c132e40066a7
 github.com/google/shlex                             6f45313302b9c56850fc17f99e40caebce98c716
 github.com/opentracing-contrib/go-stdlib            b1a47cfbdd7543e70e9ef3e73d0802ad306cc1cc
 github.com/mitchellh/hashstructure                  2bca23e0e452137f789efbc8610126fd8b94f73b
 github.com/gofrs/flock                              7f43ea2e6a643ad441fc12d0ecc0d3388b300c53 # v0.7.0
+github.com/containerd/go-cni                        5a4663dad645d9736aa62b366c8a7a07dbe751cc # master
+github.com/containernetworking/cni                  142cde0c766cd6055cc7fdfdcb44579c0c9c35bf # somewhere between v0.6.0 and v0.7.0-alpha0
```

Comment:
two new dependencies coming from buildkit

@tonistiigi was this intentional (as in; are these needed for buildkit's use in docker, or could we move some code to get rid of the dependencies?)

Reply:
you only get them cause you are vendoring buildkit that is older than master(and older than current vendor)

Reply:
hmmm; I think I vendored from your PR https://github.com/moby/buildkit/pull/1107 or is that outdated?